packages/patternfly-4/react-core/src/components/Expandable/Expandable.tsx

```diff
@@ -38,12 +42,24 @@ export class Expandable extends React.Component<ExpandableProps, ExpandableState
   static defaultProps = {
     className: '',
     toggleText: '',
+    toggleTextExpanded: '',
+    toggleTextCollapsed: '',
     onToggle: (): any => undefined,
     isFocused: false,
     isActive: false,
     isHovered: false
   };
 
+  private calculateToggleText(toggleText: string, toggleTextExpanded: string, toggleTextCollapsed: string, propOrStateIsExpanded: boolean) {
+    if (propOrStateIsExpanded && toggleTextExpanded !== '') {
+      return toggleTextExpanded;
+    }
+    if (!propOrStateIsExpanded && toggleTextCollapsed !== '') {
+      return toggleTextCollapsed;
+    }
+    return toggleText;
+  }
+
   render() {
     const {
       onToggle: onToggleProp,
```

Comment:
On line 77, onToggle is defined, again

`let onToggle = onToggleProp;`

Reply:
It looks like this is being done intentionally by whoever wrote that section.

Reply:
A comment would help here, but I won't hold up the PR